src/agent/src/rpc.rs

```diff
@@ -85,6 +85,8 @@ pub const CONTAINER_BASE: &str = "/run/kata-containers";
 const MODPROBE_PATH: &str = "/sbin/modprobe";
 const ANNO_K8S_IMAGE_NAME: &str = "io.kubernetes.cri.image-name";
 const CONFIG_JSON: &str = "config.json";
+const INIT_TRUSTED_STORAGE: &str = "init_trusted_storage.sh";
```

Comment:
Don't we want to have an absolute path here?

Reply:
+1 to @c3d's comment. Maybe `/usr/bin/kata-init-trusted-storage` or similar? Also, I'd consider removing the `.sh` suffix to allow users to create their own version of this file (in their custom rootfs) but writing it in rust, say.